packages/iris-grid/src/IrisGridTheme.module.scss

```
@@ -44,6 +44,7 @@ $header-height: 30px;
   scroll-bar-active-color: $gray-300;
 
   text-color: $white;
+  hyper-link-color: $primary;
```

Hyperlink is one word
```suggestion
  hyperlink-color: $primary;
```